Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "iso8601"
-version = "0.3.1-alpha.0"
+version = "0.3.1-alpha.1"
```

Comment:
Please don't change this version number. We will do so as part of the release process.